doc/manifest-tool-guide.md

```diff
@@ -0,0 +1,92 @@
+# Manifest Tool Guide
+
+This guide explains how to setup a validator so the key pairs used to sign and
+verify validations may safely change. This procedure does not require manual
+reconfiguration of servers that trust this validator.
+
+Validators use two types of key pairs: *master keys* and *ephemeral
+keys*. Ephemeral keys are used to sign and verify validations. Master keys are
+used to sign and verify manifests that change ephemeral keys. The master secret
+key should be tightly controlled. The ephemeral secret key needs to be present
+in the config file.
+
+## Validator Keys
+
+When first setting up a validator, use the `validator-keys` tool to generate a
+master key pair:
+
+```
+  $ validator-keys create_master_keys
+```
+
+Sample output:
+```
+  Master validator keys stored in /home/ubuntu/.ripple/validator-keys.json
+```
+
+Sample key file:
+```
+  {
+     "key_type" : "ed25519",
+     "master_secret" : "paUvjG1z51cpSxL3Wc9DaGTEogYk4AEesLJJaawBcVQjejxGFcb",
+     "sequence" : 4294967295,
+     "validation_public_key" : "nHUtNnLVx7odrz5dnfb2xp1gbEeJPbzJWfdicSkGyVw1eE5GpjQr"
+  }
+```
+
+The `validation_public_key` is the master public key. Any other rippled trusting
+the validator needs to add the master public key to its config. Only add keys
+received from trusted sources.
+
+The `master_secret` is the corresponding master secret key. **DO NOT SHARE THIS
+KEY**. The master secret key will be used to sign manifests that change
+ephemeral validation keys. Put the master secret key in a secure but recoverable
+location.
+
+## Validation Keys
+
+When first setting up a validator, or when changing the ephemeral keys, use the
+`validator-keys` tool to create a new ephemeral key pair:
+
+```
+  $ validator-keys create_signing_keys
```

Comment:
I'm happy to defer to the professional, @mduo13, on naming issues like this.

I don't feel strongly about this, but I have an inclination to use either "validator" or "validation" consistently in all the naming and the JSON keys rather than flipping between "validation" and "validator".  I suspect at least some of the JSON keys are already cemented into "validation", so we might as well stick with that.  If we stick consistently with "validation" that's one less thing for a user to think about.  And if a user resorts to grepping then it's just one chunk of text they have to grep for.  I don't think "validation" is any less informative than "validator".  If we're not cemented into "validation" then I just have the inclination that we use "validator" or "validation" consistently, at least in this subset of the world.

The hard thing for us developers to remember is that users don't use this stuff very often.  We have our heads submerged in the details as we code it up.  A user wants to get in and out in a hurry (at least on stuff like this).  Everything we can do to minimize what they have to think about will at least have them cursing us less.  Just my two pesos.